sdk/monitor/opentelemetry-exporter/samples/basic-tracer-node/README.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+This example shows how to use [@opentelemetry/tracing](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-tracing) to instrument a simple Node.js application - e.g. a batch job.
+
+Our example will export spans to Application Insights (Azure Monitor)
+
+## Installation
+
+```sh
+$ # from this directory
+$ npm install
+```
+
+## Run the Application
+
+```sh
+$ # from this directory
+$ npm start
+```
+
+## Useful links
+- For more information on OpenTelemetry, visit: <https://opentelemetry.io/>
+- For more information on tracing, visit: <https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-tracing>
+
+## LICENSE
+
+Apache License 2.0
+
+[opentelemetry-collector-url]: https://github.com/open-telemetry/opentelemetry-exporter-collector
```

Comment:
this link doesn't seem to be used above?

Reply:
Removed